setup.py

```diff
@@ -43,4 +43,8 @@
     ],
     license="GPLv3+",
     python_requires=">=3.6",
+    setup_requires=['setuptools-git-versioning'],
+    version_config={
+        "dirty_template": "{tag}.rc{ccount}-git.{sha}",
```

Comment:
This is giving us version in form: `sprint-25.rc2-git.5ec90cf7`, there's a difference between this and what was suggested in jira story: `sprint25.rc1+git.77269de5`. When trying out the plus sign before git in the template, it gives me a dash instead of a plus as a result. Do you think it's ok to have a dash here?

Reply:
Doesn't matter IMO, it's not going to be a valid PyPI version either way (and that's fine).

BTW, what exactly is the difference between `dirty_template` and `dev_template`?

Reply:
The template is reflecting the state of repo. It will first check which template to use and then if there is `version_config` for that template it will change the string accordingly. I'll change it to `dev_template` because `dirty_template` is expecting unstashed changes (which was case for my local repo).

Reply:
It wasn't working for me until I found out that `dirty_template` also applies if you have untracked files. When I fixed that, everything worked as expected :+1: